src/ResourceManager/HDInsight/Commands.HDInsight/JobCommands/GetAzureHDInsightJobOutputCommand.cs

```diff
@@ -133,25 +134,38 @@ private static string Convert(Stream stream)
 
         internal IStorageAccess GetDefaultStorageAccess(string resourceGroupName, string clusterName)
         {
+            var StorageAccountSuffix = "";
             if (DefaultContainer == null && DefaultStorageAccountName == null && DefaultStorageAccountKey == null)
             {
-                var DefaultStorageAccount = GetDefaultStorageAccount(resourceGroupName, clusterName);
-
+                var DefaultStorageAccount = GetDefaultStorageAccount(resourceGroupName, clusterName);              
                 var wasbAccount = DefaultStorageAccount as AzureHDInsightWASBDefaultStorageAccount;
                 if (wasbAccount != null)
                 {
                     DefaultContainer = wasbAccount.StorageContainerName;
                     DefaultStorageAccountName = wasbAccount.StorageAccountName;
                     DefaultStorageAccountKey = wasbAccount.StorageAccountKey;
+
```

Comment:
This is an existing cmdlet and I am not modfying the parameters/arguments to it. I think the existing tests should suffice.